### ISSUE TITLE
test-template.enex not found runtime error

### ISSUE THREAD

**Issue by kkpan11** [akosbalasko/yarle#452]
Hi there,

I found out the default enex sources settings are included when changing the output format.

## Default Settings:
- config.json:
  ```
  enexSources: [
	  "./test-template.enex"
  ],
  ```
- src/yarle.ts:
  ```
  enexSources: ["notebook.enex"],
  ```

## Reproduce Steps:
1. Launch app.
2. Choose enex files with the file selection dialog.
3. Change output format from default `Obsidian` to `LogSeq` or `Other Standard Markdown-based application`.
4. Choose an output directory.
5. Then the `test-template.enex` will be included in enex sources files.

---
So I made some modifications in the following:
1. Remove the **enexSources** element of `config.json`, the `notebook.enex` will be included in enex sources files which is coming from `src/yarle.ts`.
2. After removing the **enexSources** element of `src/yarle.ts`, the result is becoming correct.

I was wondering if the initial `enexSources` settings are removed, will there be any effects?

**Comment by akosbalasko** [comment]
Hi @kkpan11 !
Thank you for the hint, I missed this wrong behavior. I wouldn't remove anything from config.json or yarle's default config in the ts file, because the main conversion can be triggered without UI as well, therefore I'd like to provide an example for the proper configuration. 
But the root cause of the problem what you pointed out was related to the outputFormat change action, which updated the config file together with the enex file and the output folder as well. 
I fixed the problem in PR mentioned below, and I'll keep you posted when it is release (still today). Could you please confirm that the fix works well on your side too? 

Thanks again!

**Comment by akosbalasko** [comment]
Hi @kkpan11 ,

It's out in version 5.0.7 .

Thank you for raising this issue up, and for the clear reproduce steps, I really appreciate them! 
And of course, do not hesitate to contact if you face with any issues!

**Comment by kkpan11** [comment]
Hi @akosbalasko,

Thank you for the clarification. :)

And the problem is fixed with the new version `5.0.7`.
Thanks for the update quickly.
